pom.xml

```diff
@@ -82,7 +82,7 @@
 
     <sonar.version>9.4.0.54424</sonar.version>
     <sonarlint.plugin.api.version>6.3.0.36253</sonarlint.plugin.api.version>
-    <analyzer.commons.version>1.24.0.969</analyzer.commons.version>
```

Comment:
TODO: update this version to a one from master...

Reply:
PR on commons merged!

Reply:
I updated the version to rely on `1.24.0.973`.